### PR TITLE
KOGITO-9492 Update bump-version script

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-9492

- `image.yaml` was not correctly updated
- bundle csv use Major.minor instead of full version
- CI will publish the Major.minor image

**Description of the change:**

**Motivation for the change:**

**Checklist**

- [ ] Add or Modify a unit test for your change
- [ ] Have you verified that tall the tests are passing?

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Comment by radtriste** [comment]
Forgot to push ci changes. Will push it on Sunday/Monday.

**Review by spolti — APPROVED** [review]

File: .ci/jenkins/Jenkinsfile.promote [hunk 1]
```diff
@@ -62,6 +62,12 @@ pipeline {
                         // Tag with given parameter tag
                         promoteImage(helper.getImageFullTag(imageName, baseImageParamsPrefix), helper.getImageFullTag(imageName, promoteImageParamsPrefix))
 
+                        try {
+                            promoteImage(helper.getImageReducedTag(imageName, baseImageParamsPrefix), helper.getImageReducedTag(imageName, promoteImageParamsPrefix))
```
Comment (radtriste) [inline .ci/jenkins/Jenkinsfile.promote:66]:
```suggestion
                            promoteImage(helper.getImageFullTag(imageName, baseImageParamsPrefix), helper.getImageReducedTag(imageName, promoteImageParamsPrefix))
```

File: .ci/jenkins/Jenkinsfile.promote [hunk 2]
```diff
@@ -62,6 +62,12 @@ pipeline {
...
 
+                        try {
+                            promoteImage(helper.getImageFullTag(imageName, baseImageParamsPrefix), helper.getImageReducedTag(imageName, promoteImageParamsPrefix))
+                        } catch (err) {
+                            echo 'Reduced tag cannot be applied'
+                        }
+
                         // Tag with `latest` tag if asked for as parameter
                         if (helper.isDeployLatestTag()) {
                             promoteImage(helper.getImageFullTag(imageName, baseImageParamsPrefix, 'latest'), helper.getImageFullTag(imageName, promoteImageParamsPrefix, 'latest'))
```
Comment (radtriste) [inline .ci/jenkins/Jenkinsfile.promote:73]:
```suggestion
                             promoteImage(helper.getImageFullTag(imageName, baseImageParamsPrefix), helper.getImageFullTag(imageName, promoteImageParamsPrefix, 'latest'))
```

**Comment by radtriste** [comment]
Jenkins retest this

**Pull request merged**